src/styles/header.scss

```diff
@@ -0,0 +1,61 @@
+.logo {
+  margin-top: 4px;
+}
```

Comment:
Don't add external styles (positioning or margins) to BEM-blocks. Use mix where necessary and move all external styles under element selector.

Reply:
you can make the logo a headers element